create_pr.sh

```
@@ -0,0 +1,106 @@
+if git push -u origin $BRANCH_NAME; then
+  echo -e "${GREEN}Branch pushed successfully${NC}"
+else
+  echo -e "${RED}Failed to push branch${NC}"
+  echo -e "${YELLOW}You can still continue with PR creation for local changes${NC}"
+fi
+
+# Step 5: Set default repository for GitHub CLI if needed
+if command -v gh &>/dev/null; then
+  if ! gh repo view &>/dev/null; then
+    echo -e "\n${YELLOW}Setting default repository for GitHub CLI...${NC}"
+    gh repo set-default $(git config --get remote.origin.url | sed 's/.*github.com[:\/]\(.*\)\.git/\1/' 2>/dev/null || echo "")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)